### ISSUE TITLE
Signature error

### ISSUE THREAD

**Issue by procw** [MolecularMatters/psd_sdk#7]
Sorry, I fixed path error, but Signature error raised.
```
<?xpacket end="w"?>

***ERROR*** [ImageResources] Image resources section seems to be corrupt, signature does not match "8BIM".Program ended with exit code: 0
```
how can I solve it?
![Screen Shot 2021-07-15 at 11 31 13](https://user-images.githubusercontent.com/57867146/125718768-22d7f4d6-248e-4cab-9b72-10fcc13f5e7d.png)

_Originally posted by @PROCW in https://github.com/MolecularMatters/psd_sdk/issues/6#issuecomment-880341665_

**Comment by tivolo** [comment]
This could be caused by several things:
1) The reader was confused by preceeding data, so when it expects the resource identifier, something else is read from the data stream.
2) An endianness error.
3) Some new kind of PSD format that the reader cannot handle yet.

What is the actual data read from the stream?

**Comment by procw** [comment]
Okay, Thanks.
But I build with sample psd file but I don't see result.
What is the end result of building this sdk?
Are you displaying the psd file in the view?
Thanks for your help.

**Comment by MolecularMatters** [comment]
Running the sample will read Sample.psd and output a few .TGA files for the individual layers to demonstrate that they have been loaded correctly.
The sample will also output a few .PSD files to demonstrate the writing capabilities of the SDK.

Regarding your error about corrupted image resources, maybe @oluseyi can chime in?
They provided the macOS port of the SDK.

**Comment by oluseyi** [comment]
@PROCW can you give me the context here? This issue isn't linked to anything else, and I haven't looked at this repo/codebase in months; what are you trying to do, and what's happening?

**Comment by procw** [comment]
Thanks for your reply,
But Can I see the psd file on view instead of .TGA files?
And I added the link for reading error psd file.
https://we.tl/t-GWlt8IQiKW

**Comment by playmer** [comment]
Hey @PROCW, can you give the above PR a try? I can't promise anything else, but I'm hoping it fixes the error you're seeing.

**Comment by MolecularMatters** [comment]
I just fixed this in 91643c3cdd42e4288bd56590407c42d54d450f08.
@PROCW, please give this a try.

**Comment by MolecularMatters** [comment]
Closing this for now, as this no longer seems to be an issue.
